tests/e2e/can-be-deleted/page_objects.py

```
@@ -0,0 +1,366 @@
+    def navigate(self):
+        """Navigate to the devices page."""
+        self.page.goto(f"{self.base_url}/devices")
+        self.page.wait_for_load_state("networkidle")
+
+    def click_add_device(self):
+        """Click the add device button."""
+        add_button = self.page.get_by_role("button", name="Add")
+        add_button.click()
+
+    def fill_device_form(
+        self, device_type: str, name: str, ip: str, pixel_count: int
```

Ruff (0.14.10)

Unused method argument: `device_type`

(ARG002)